packages/test/test-service-load/testConfig.json

```diff
@@ -37,6 +37,11 @@
 						"Fluid.Container.summarizeProtocolTree2": [true, false],
 						"Fluid.ContainerRuntime.Test.SummarizationRecoveryMethod": ["restart", "default"]
 					}
+				},
+				"tinylicious": {
+					"configurations": {
```

Comment:
I believe we need to move it from here into a location that applies to all services (I think it's in some .ts file).